### PR TITLE
PMM-7: fix lint errors

### PR DESCRIPTION
**Pull request by ritbl**
PMM-7

### CODE REVIEW

File: admin/commands/inventory/list_agents.go [hunk 1]
```diff
@@ -69,7 +71,7 @@ type listAgentsResult struct {
 
 func (a listResultAgent) NiceAgentStatus() string {
 	res := a.Status
-	res = strings.Title(strings.ToLower(res))
```
Comment (ritbl) [inline admin/commands/inventory/list_agents.go:72]: deprecated

File: agent/agents/mysql/perfschema/perfschema.go [hunk 2]
```diff
@@ -338,7 +338,7 @@ func (m *PerfSchema) getNewBuckets(periodStart time.Time, periodLengthSecs uint3
 					b.Common.IsTruncated = truncated
 				}
 				b.Common.Example = example
-				b.Common.ExampleFormat = agentpb.ExampleFormat_EXAMPLE
+				b.Common.ExampleFormat = agentpb.ExampleFormat_EXAMPLE // nolint:staticcheck
```
Comment (ritbl) [inline agent/agents/mysql/perfschema/perfschema.go:341]: I think it is safer for now to ignore

**Review by BupycHuk — APPROVED** [review]

**Review by michal-kralik — APPROVED** [review]

File: agent/agents/postgres/pgstatmonitor/pgstatmonitor.go [hunk 3]
```diff
@@ -211,7 +211,7 @@ func (m *PGStatMonitorQAN) Run(ctx context.Context) {
 	if current, _, err := m.monitorCache.getStatMonitorExtended(ctx, m.q, normalizedQuery); err == nil {
 		m.monitorCache.refresh(current)
 		m.l.Debugf("Got %d initial stat monitor.", len(current))
-		running = true
+		running = true // nolint: ineffassign
```
Comment (artemgavrilov) [inline agent/agents/postgres/pgstatmonitor/pgstatmonitor.go:214]: I think that linter warning actually highlights the problem. That variable overwritten on line 255.

File: agent/utils/depstests/protobuf_test.go [hunk 4]
```diff
@@ -18,7 +18,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/golang/protobuf/jsonpb"
+	"github.com/golang/protobuf/jsonpb" //nolint:staticcheck
```
Comment (artemgavrilov) [inline agent/utils/depstests/protobuf_test.go:21]: I think that one was also a good warning

File: managed/services/management/azure_database.go [hunk 5]
```diff
@@ -21,7 +21,7 @@ import (
 	"fmt"
 
 	"github.com/AlekSi/pointer"
-	"github.com/Azure/azure-sdk-for-go/services/resourcegraph/mgmt/2019-04-01/resourcegraph"
+	"github.com/Azure/azure-sdk-for-go/services/resourcegraph/mgmt/2019-04-01/resourcegraph" // nolint:staticcheck
```
Comment (artemgavrilov) [inline managed/services/management/azure_database.go:24]: And that one

File: agent/agents/postgres/pgstatmonitor/pgstatmonitor.go [hunk 6]
```diff
@@ -473,7 +473,7 @@ func (m *PGStatMonitorQAN) makeBuckets(current, cache map[time.Time]map[string]*
 
 			if !m.disableQueryExamples && currentPSM.Example != "" {
 				mb.Common.Example = currentPSM.Example
-				mb.Common.ExampleFormat = agentpb.ExampleFormat_EXAMPLE
+				mb.Common.ExampleFormat = agentpb.ExampleFormat_EXAMPLE // nolint:staticcheck
```
Comment (artemgavrilov) [inline agent/agents/postgres/pgstatmonitor/pgstatmonitor.go:476]: Can you please create ticket for that. Otherwise we will forget about that problem.

**Review by artemgavrilov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**